src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+import { forecasts } from "../../data/forecast.json";
```

Comment:
Instead of importing some data to be used in a test, it would be better to declare it as a valid props object in the file itself for better visibility. I know that might sound like you're going against DRY principles, but in testing thats actually okay as you want to have visibility in each file of what test values are being added for each component